mars/services/task/supervisor/processor.py

```diff
@@ -456,3 +458,21 @@ def _finish(self):
 
     def is_done(self) -> bool:
         return self.done.is_set()
+
+    async def remove_func_storage(self):
+        tileables_need_clean_up_func = [
+            self.tileable_id_to_tileable[tileable_id]
+            for tileable_id in self._preprocessor.tileable_ids_need_clean_up_func
+        ]
+        func_keys = [t.op.func_key for t in tileables_need_clean_up_func]
+        logger.debug(f"Remove func storage {func_keys}")
+        ctx = get_context()
+        # Note: Considering that remove_func_storage is executed while TaskProcessor is being
+        # destroyed, creating storage_api may fail. Error type that may occur is still under
+        # investigation as well as experiment.
+        storage_api = await StorageAPI.create(ctx.session_id, ctx.supervisor_address)
```

Comment:
`function` are put into storage by `ctx.storage_put`, why not cleanup func by `ctx.storage_delete`